README.md

```diff
@@ -1 +1,192 @@
+
+## 🛠 Built With <a name="HTML and CSS"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[A nice heading]**
+- **[Loads very fast]**
+- **[Responsive to different devices]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+visit my github
+check Hello-Microvese Project repo
+
+### Setup
+
+Clone this repository to your desired folder:
+cd hello-world
+
+
+
+### Install
+
+Install this project with:
+
+
+  cd Hello-world
+
+
+### Usage
+
+To run the project, execute the following command:
+
+sh hello-microverse project
+
+### Run tests
+
+To run tests, run the following command:
+
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Misikir Delelew**
+
+- GitHub: [@misikirdelelw](https://github.com/MishMD)
+- Twitter: [@misikirdelelw](https://twitter.com/misikir-delelew)
+- LinkedIn: [@misikirdelelew](https://linkedin.com/in/misikir-delelew-114a5946)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **[Footer at the bottom]**
+- [ ] **[Logo at the top left]**
+- [ ] **[navigation bar]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project reach me through github and I will be
+very happy to connect with you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank for microverse team who has prepared the configuration document.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] I have noticed that MIT link is broken, Kindly ensure that you add the `MIT`( lincense) file. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose.
**Key note: ** Kindly commit add the live MIT in the `feature` branch to enable other code reviewers track it easily.